source/user-manual/configuring-cluster/index.rst

```diff
@@ -1,13 +1,13 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. meta::
+  :description: Learn more about the cluster node configuration in Wazuh: labels, Filebeat, Splunk forwarder installation, and more. 
+
 .. _configuring-cluster:
 
 Deploying a Wazuh cluster
 ===========================
 
-.. meta::
-  :description: A complete user manual about how to configure a Wazuh cluster.
```

Comment:
This meta description is way too short. We need a meta description between 120 and 150 characters, @MarVergara